README.rst

```diff
@@ -229,3 +229,30 @@ Appendix B: Adding No-Op Stuff to Test Sandbox Deploys
 ----------------------------------------------------------
 
 Let's try this.
+
+
+Subscriptions
+-------------
+
+This payment MFE has introduced an additional React route, ``payment.stage.edx.org/subscription``, specifically designed to handle recurring payments. Currently, we are only permitting one payment method to manage recurring purchases, namely stripe subscriptions.
```

Comment:
Can you specify `handle B2C recurring payments`

Reply:
https://2u-internal.atlassian.net/wiki/spaces/RS/pages/551321788/How+to+handle+b2c+recurring+payments+on+the+payment+checkout+page